src/WebJobs.Script/IHostOptionsProvider.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Extensions.DependencyInjection;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public interface IHostOptionsProvider
+    {
+        string GetPayload();
```

Comment:
Rename to GetOptions. Might be better to have this return a JObject as well for versatility. Caller can format as needed.

Reply:
Done